test/e2e/monitoring_stack_controller_test.go

```diff
@@ -973,6 +973,7 @@ func newService(t *testing.T, name, namespace string, labels, selector map[strin
 }
 
 func newPrometheusPod(t *testing.T, name, ns string) *corev1.Pod {
```

Comment:
could you please rename this to `newPrometheusExamplePod` now that the definition has changed?